src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/stylesheet.css

```diff
@@ -727,7 +727,6 @@ div.checkboxes > label > input {
 }
 .col-first, .col-second, .col-constructor-name {
     vertical-align:top;
-    overflow: auto;
```

Comment:
Is this actually necessary, and if so, where exactly? It seems that the elements affected by this change already have the following applied, which should have the same effect:

    .summary-table > div {
        overflow: auto hidden;
    }

Reply:
API doc(.html) generated using this stylesheet.css didn't had overflow: auto applied from .summary-table > div element but overflow been applied from .col-first, .col-second, .col-constructor-name which caused our achecker failure. .summary-table doesn't have affect.

Please find the below stylesheet.css used in one of our javadoc generated API doc
```
.summary-table > div, .details-table > div {
    text-align:left;
    padding: 8px 3px 3px 7px;
}
.col-first, .col-second, .col-last, .col-constructor-name, .col-summary-item-name {
    vertical-align:top;
    padding-right:0;
    padding-top:8px;
    padding-bottom:3px;
}
.table-header {
    background:#dee3e9;
    font-weight: bold;
}
.col-first, .col-first {
    font-size:13px;
}
.col-second, .col-second, .col-last, .col-constructor-name, .col-summary-item-name, .col-last {
    font-size:13px;
}
.col-first, .col-second, .col-constructor-name {
    vertical-align:top;
    overflow: auto;
}
```

Reply:
The current stylesheet has the following rule at line 688:

```
.summary-table > div, .details-table > div {
    text-align:left;
    padding: 8px 3px 3px 7px;
    overflow: auto hidden;
    scrollbar-width: thin;
}
```

Granted, the `overflow` declaration was added relatively recently (October or last year). Does the problem still persist with this?

Reply:
As you mentioned earlier, overflow:auto at line 730 is getting overlapped by line 688. Still we see achecker failing with `element_scrollable_tabbable` violation. As summary-table is a grid, I think overflow may not have impact. Please correct me. Thank you.

Reply:
@hns Could you please validate my comment. Thank you.

Reply:
After having spent spent some time researching the problem and the proposed solution, I find that the premise of this issue is wrong. With current versions of supported browsers on my computer (Firefox, Chrome, Safari) the scrollable tabs can be focused using the tab key (in Safari the "press tab to highlight items" option has to be enabled in the advanced settings). It may be that this is a new addition in Chrome and Safari, and that they did not support this at the time when the issue was detected and filed.

Apart from that, I don't think that the proposed change would be a good fix for the problem. By removing the `overflow: auto` declarations in the given `<div>s`, overflow becomes invisible, or overflows the layout in the case of the rightmost column. Making content inaccessible for all users is not a good solution for the problem of content being inaccessible to keyboard users, and breaking the layout of the page is not desirable either.

Given that mainstream browsers seem to have followed Firefox and converted on making scrollable elements tabbable I would not consider this a problem in our documentation.